xbmc/guilib/GUIAudioManager.cpp

```diff
@@ -160,64 +164,35 @@ void CGUIAudioManager::PlayPythonSound(const std::string& strFileName, bool useC
     return;
 
   // If we already loaded the sound, just play it
-  pythonSoundsMap::iterator itsb=m_pythonSounds.find(strFileName);
+  const auto& itsb = m_pythonSounds.find(strFileName);
   if (itsb != m_pythonSounds.end())
   {
-    IAESound* sound = itsb->second;
+    const auto& sound = itsb->second;
```

Comment:
no ref as it's a pointer?

Reply:
But because it's a `std::shared_ptr` a copy would perform reference counting. I suggest to keep it as is if it isn't to confusing :)